my-app/src/app/app-routing.module.ts

```diff
@@ -1,13 +1,13 @@
 import {NgModule} from '@angular/core';
 import {Routes, RouterModule} from '@angular/router';
-import {HomeComponent} from './home/home.component';
+import {BlogComponent} from './blog/blog.component';
 
 
 const routes: Routes =
   [
     {
       path: '',
-      component: HomeComponent
+      component: BlogComponent
```

Comment:
I see you took out Home Component and replaced it with Blog Component, but how will people get to the Homepage now?